src/functionalTest/resources/features/common/data-store/FT_NoCCaseType/snippets/NoC_Case_Data2.td.json

```diff
@@ -0,0 +1,47 @@
+{
+  "_guid_": "NoC_Case_Data_2",
```

Comment:
If this is used:

> the updated ACA-68 has now updated the original `NoC_Case_Data.td.json` to match the same data source.  So it could be replaced by: 

* https://github.com/hmcts/aac-manage-case-assignment/blob/ACA-68/src/functionalTest/resources/features/common/data-store/FT_NoCCaseType/snippets/NoC_Case_Data.td.json
* https://github.com/hmcts/aac-manage-case-assignment/blob/ACA-68/src/functionalTest/resources/features/common/data-store/FT_NoCCaseType/snippets/NoC_Case_Data__publicFields.td.json

Otherwise:
> delete